src/quo2/components/list_items/channel.cljs

```diff
@@ -1,12 +1,11 @@
 (ns quo2.components.list-items.channel
   (:require [quo2.components.avatars.channel-avatar :as channel-avatar]
+            [quo2.components.common.unread-grey-dot.view :refer [unread-grey-dot]]
             [quo2.components.counter.counter :as quo2.counter]
             [quo2.components.icon :as quo2.icons]
             [quo2.components.markdown.text :as quo2.text]
             [quo2.foundations.colors :as colors]
-            [quo2.theme :as theme]
-            [react-native.core :as rn]
-            [quo2.components.common.unread-grey-dot.view :refer [unread-grey-dot]]))
+            [react-native.core :as rn]))
```

Comment:
Sorted requires and stopped using `quo2.theme`

Reply:
actually I think you need to use `quo2.theme` here. Ideally anywhere we use `colors/theme-colors` function the 3rd parameter `theme` will be passed. Eventually we will have to do this here anyway so you may as well do it now.
i.e wrap this component in `with-theme` HOC so that it has access to `theme` prop and then pass it as a param to `theme-colors` function.

Reply:
Thank you so much for noticing this @J-Son89 !

I'd prefer to address it in  #16461, since that issue is going to refactor and check the styles, I updated this issue to mention it. :+1: